historical_system_profiles/notification_service_interface.py

```diff
@@ -56,6 +56,8 @@ class _NotificationEventBase:
     """Base event on which all notification events are built"""
 
     def __init__(self, event_type, account_id, context={}):
+        self.notification_bundle = config.notification_bundle
+        self.notification_app = config.notification_app
```

Comment:
Good catch.  Also remove the unnecessary lines in NotificationServiceInterface lines 20-21.

Reply:
We still need them for lines 27-28.

Reply:
Ah, right.